spring-cloud-dataflow-app-launcher/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM anapsix/alpine-java:8
```

Comment:
It's the most lightweight and stable. @markpollack has spend some time researching it.